### ISSUE TITLE
Keyword-spacing rule incorrectly applied to `default` keyword

### ISSUE THREAD

**Issue by teddy-error** [eslint/eslint#11096]
**Tell us about your environment**

* **ESLint Version:** 5.7.0
* **Node Version:** 10.10.0

**What parser (default, Babel-ESLint, etc.) are you using?**
Default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    "env": {
        "browser": true,
        "es6": true,
        "node": true
    },
    "extends": "eslint:recommended",
    "parserOptions": {
        "ecmaVersion": 2018,
        "sourceType": "module"
    },
    "rules": {
        "keyword-spacing": "error",
    }
};
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const abc = [1, 2, 3];
export default{ abc };
```

**What did you expect to happen?**
An error should occur since there is no space after the `default` keyword, but it does not.

**Are you willing to submit a pull request to fix this bug?**
I am not yet familiar with the source but I suspect this is caused by [how export default keyword parsing is handled](https://github.com/eslint/eslint/blob/master/lib/rules/keyword-spacing.js#L557) – `export default` is two tokens rather than one and yet it seems that only the `export` keyword is evaluated. For some reason, exportDefault is treated differently from exportAll and exportName

**Comment by Aladdin-ADD** [comment]
I was able to repro it.
